prototype.py

```diff
@@ -1,22 +1,95 @@
+from datetime import datetime
+from multiprocessing import Pool
+import json
 import os
 import io
+from itertools import islice
 import re
 import csv
 import requests
+import pandas as pd
 
+session = requests.session()
+if os.getenv('WITH_TOR') is not None:
+  session.proxies = {'http':  'socks5://127.0.0.1:9050', 'https': 'socks5://127.0.0.1:9050'}
 
+POOL_SIZE = int(os.getenv('POOL_SIZE', 8))
 DOCTOLIB_HEADERS = {
-    'X-Covid-Tracker-Key': os.environ.get('DOCTOLIB_API_KEY', None)
+    'X-Covid-Tracker-Key': os.environ.get('DOCTOLIB_APIKEY', None)
```

Comment:
Normalement le secret rentré par Guillaume s'appelle bien comme ça :

```suggestion
    'X-Covid-Tracker-Key': os.environ.get('DOCTOLIB_API_KEY', None)
```